lib/finsta/posts.ex

```diff
@@ -88,6 +94,9 @@ defmodule Finsta.Posts do
   """
   def delete_post(%Post{} = post) do
     Repo.delete(post)
+    |> tap(fn {:ok, post} ->
+      Phoenix.PubSub.broadcast(Finsta.PubSub, "posts_topic", {:delete, post})
+    end)
```

Comment:
Since we are using this same broadcast pattern in three different functions, what do you think about having one private function to handle it? something like:

```elixir
defp broadcast_post({:error, changeset}, _message_type), do: {:error, changeset}

defp broadcast_post({:ok, post}, message_type), do: Phoenix.PubSub.broadcast(Finsta.PubSub, "posts_topic", {message_type, post})
```